### DISCUSSION TITLE
Report wrong linked file

### DISCUSSION THREAD

**Discussion by efcjimbo** [elrumo/macOS_Big_Sur_icons_replacements#1401] [Help with the website]
I've come across this a few times. Download an icon and get totally different icon. How do you report a bad link.

Adobe Photoshop
user [jevr0oz]
on
28/07/2022

is not downloading the correct file

**Comment by 4uto3lend** [comment]
[Arc Browser Light](https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/27bc7aae85440722f3033a24ba7ca00a_1685538243230.icns) from [maxiflux](https://macosicons.com/#/u/maxiflux) as well.
